sdk/contentsafety/Azure.AI.ContentSafety/tsp-location.yaml

```diff
@@ -1,4 +1,4 @@
-commit: 93cd7838702286b12a0cd952d83369c7af612da3
+commit: 4429b7e6c91faeb560cfd9cdc57091565493ff7a
```

Comment:
updated to a feature branch commit which only add `@convenientAPI(false)` to patch operations